### DISCUSSION TITLE
How to make YT scanner crawl deeper in the past

### DISCUSSION THREAD

**Discussion by alfonder** [umputun/feed-master#116] [Q&A]
On creating a new item in yt section I defined "keep: 5". But then I realized that it'd be better to have more last episodes. I changed config to "keep: 10" and restarted container. (BTW I don't know if it's possible to apply changed config without restarting). I see in the log, that the option is active:
`2023/02/25 15:13:27.212 [INFO]  got 15 entries for ОСНОВА, limit to 10`
But the rss for that yt channel still contains the last 5 episodes. I guess FM reserves 5 more slots for future videos and doesn't go deeper into the past. 
The question is %subj. As if I initially added the channel with "keep: 10" parameter.

**Comment by umputun** [comment]
the already processed entries won't be added, i.e. setting `keep: 10` will work for the future entries only. In other words, eventually it will have 10 items, but you need to wait till 5 more posted on youtube

If you don't care about the current feeds and all you want is to reset to 10 - you can stop the container, remove all var and start it. This will start from the clear state and will download 10 items.

**Reply by alfonder** [reply]
Thank you for the answer. But I don't want to lose the other feeds. I suppose that FM identifies YT feeds by the channel ID, therefore renaming the feed won't help. Is it correct?
